src/IssueViz/Editor/SelectedIssueTagging/Adornment/IssueLocationAdornment.cs

```diff
@@ -28,37 +28,42 @@ namespace SonarLint.VisualStudio.IssueVisualization.Editor.SelectedIssueTagging.
     internal class IssueLocationAdornment : Button
     {
         private readonly TextBlock textBlock;
-        public IAnalysisIssueLocationVisualization Location { get; }
+        public IAnalysisIssueLocationVisualization LocationViz { get; }
 
-        public IssueLocationAdornment(ISelectedIssueLocationTag issueTag, IFormattedLineSource formattedLineSource)
+        public IssueLocationAdornment(IAnalysisIssueLocationVisualization locationViz, IFormattedLineSource formattedLineSource)
         {
             // We can't store the formatted line source since it might change
             // e.g. if the user changes the font size
-            Location = issueTag.Location;
+            LocationViz = locationViz;
 
             Margin = new System.Windows.Thickness(1, 0, 1, 0); // Space between this UI element and the editor text
             Padding = new System.Windows.Thickness(0);  // Space between the side of the control and its content    
 
             Background = Brushes.Pink;
-            BorderBrush = Brushes.Red;
+            BorderBrush = Brushes.LightCoral;
```

Comment:
Unrelated: minor visual tweak so the contrast between the border and background is less harsh.